ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicySeverity.tsx

```diff
@@ -79,8 +108,8 @@ function ViolationsByPolicySeverity() {
         CRITICAL_SEVERITY: 0,
     };
     let totalCount = 0;
-    severityCounts.forEach(({ severity, count }) => {
-        counts[severity] = parseInt(count, 10);
+    Object.entries(alertCountData ?? {}).forEach(([severity, count]) => {
+        counts[severity] = count;
         totalCount += counts[severity];
```

Comment:
I think this is a case where a reduce would be more functional, but also make an already confusing database optimization even harder to follow.

Reply:
Maybe if only computation of total, but not with assignment to `counts` is a side-effect.

@dvail For my understanding, is the assignment still needed now that the GraphQL query seems to return `AlertCounts` type which is same as `counts` object?

Reply:
Ah sorry I missed this comment since I had automerge on and was focused on the CI failures. Yeah I think the assignment is unnecessary after the change and the sum could be changed to a one line reducer like Van mentioned `Object.values(alertCountData).reduce((a, b) => a + b)`